src/plugins/contentful-ui-setting-styles-mixin.js

```diff
@@ -16,6 +16,10 @@ export default {
 			if (textAlign) {
 				customStyles = `${customStyles} text-align: ${textAlign};`;
 			}
+			// Vertical alignment of grid items
+			const verticalAlign = this.uiSetting?.dataObject?.verticalAlign ?? null;
+			customStyles = `${customStyles} align-items: ${verticalAlign};`;
```

Comment:
Please add some information on how to use this to the admin page docs alongside the other styles.